### DISCUSSION TITLE
How do I avoid being removed from Google Play?

### DISCUSSION THREAD

**Discussion by smarthjx636** [community/community#122511] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Recently (late April 2024), tens of thousands of apps have been removed from the shelves and more than a hundred have been listed every day. Let’s discuss how to prevent your app from being removed from the shelves and your account from being banned.

We work hard to write code and want to have a good development. We welcome independent developers to discuss and speak.

**Comment by smarthjx636** [comment]
Developers’ hard work, how to avoid going to waste

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @smarthjx636 !

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the `General` category. I've gone ahead and moved it for you. Good luck!
